dev-docs/adunit-reference.md

```diff
@@ -243,10 +244,38 @@ pbjs.addAdUnits({
     mediaTypes: {
         video: {
             context: 'instream',
+            plcmt: 1,
             playerSize: [640, 480],
             mimes: ['video/mp4'],
             protocols: [1, 2, 3, 4, 5, 6, 7, 8],
-            playbackmethod: [2],
+            playbackmethod: [5],
+            skip: 1
+        },
+    },
+    bids: [{
+        bidder: 'appnexus',
+        params: {
+            placementId: 13232361
+        }
+    }]
+});
+```
+
+<a name="adUnit-video-example-accompanying"></a>
+
+#### Accompanying Content
```

Comment:
@fowler446 - I may be out-of-line here, but my feedback is that these terms are confusing. 

Why isn't this header "Instream sound-off"? Having such a different phrase makes it seem like a massively different thing.

Plus, "Content" to me could mean textual content, but I think this means video content. I would suggest phrasing this "Instream video content with sound off"

Reply:
@bretg 
I'm all for making things less confusing. What I was going for here was alignment with the actual IAB definitions:
https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/develop/AdCOM%20v1.0%20FINAL.md#list_plcmtsubtypesvideo

One of which is exactly "Accompanying Content"..... which is no longer considered Instream.

Reply:
I'm not ready buckle to the IAB's poor naming choices. :-)

How about we name them reasonably and then translate to IAB-speak?

e.g.

## Player contains video content with sound off
The IAB calls this scenario "Accompanying Content". Previously, this might have been called "instream with sound-off", but as of __DATE__, the IAB has reclassified this scenario.